x-pack/qa/rolling-upgrade/src/test/resources/rest-api-spec/test/mixed_cluster/10_basic.yml

```diff
@@ -6,10 +6,10 @@
         wait_for_active_shards: all
         body:
           settings:
-# we use 1 replica to make sure we don't have shards relocating. Relocating a shard with
-# a scroll on it prevents shards from moving back into a where a scroll is running (it holds the shard lock)
-# see https://github.com/elastic/elasticsearch/issues/31827
-            number_of_replicas: 1
+# we make sure we don't have shards relocating. Relocating a shard with a scroll on it prevents shards from moving back into a where
+# a scroll is running (it holds the shard lock), see https://github.com/elastic/elasticsearch/issues/31827
+            index.routing.rebalance.enable: none
+            index.number_of_replicas: 0
```

Comment:
Did you change the number_of_replicas to 0 merely to undo the previous change or for other reasons?

Reply:
To undo the previous change, and to make it clearer what's being tested here